sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_groundedness/_groundedness.py

```diff
@@ -74,4 +92,35 @@ def __call__(
         :return: The relevance score.
         :rtype: Union[Dict[str, float], Dict[str, Union[float, Dict[str, List[float]]]]]
         """
-        return super().__call__(response=response, context=context, conversation=conversation, **kwargs)
+        if (response is None or context is None) and conversation is None:
+            msg = "Either a pair of 'response'/'context' ('query' optional) or 'conversation' must be provided."
+            raise EvaluationException(
+                message=msg,
+                internal_message=msg,
+                blame=ErrorBlame.USER_ERROR,
+                category=ErrorCategory.MISSING_FIELD,
+                target=ErrorTarget.GROUNDEDNESS_EVALUATOR,
+            )
+
+        if (query or response or context) and conversation:
+            msg = "If 'conversation' is provided, 'query', 'response', and 'context' cannot be provided."
+            raise EvaluationException(
+                message=msg,
+                internal_message=msg,
+                blame=ErrorBlame.USER_ERROR,
+                category=ErrorCategory.INVALID_VALUE,
+                target=ErrorTarget.GROUNDEDNESS_EVALUATOR,
+            )
```

Comment:
I remember Miles already added these kind of validations in the base class.

Reply:
Nvm, I saw your remove that logic from `_base_eval.py`. Is that for the user experience improvement?

Reply:
I removed them from there and added them here because his implementation didn't allow for optional parameters for specific evaluators which we now have due to the introduction of `query` in `GroundednessEvaluator`

Reply:
If we can come up with a sleek solution to amend Miles' centralized solution to incorporate edge cases of optional params, I'm happy to refactor later, but I gave it a stab and couldn't get a working adaptation quickly and since we're in a  rush to get these changes in, I decentralized it for a quick, simple fix.

Reply:
My concern is that the change you made in `_base_eval.py` would impact all the evaluators that inhert from `EvaluatorBase`. @MilesHolland to confirm

Reply:
Yes that's true. All evaluators with multiple inputs will be affected, which means that all the RAI service evals won't be smart enough to parse out query/response requirements.

That said, This _should_ be fixed once Neehar's overloading PR goes in.

Reply:
@needuv - can you chime in?

Reply:
so that PR doesn't actually change anything about how `__call__` is invoked-- the non-overloaded function is still the one that gets called, and we still need to validate that the user didn't pass every possible input

The `@overload` is just for the type-checker/intellisense to provide users with a better experience.

Reply:
I refactored Miles' original code to allow optional params. I'll remove the checks from the individual evaluators and add tests.